DistFiles/SayMore.ru.tmx

```diff
@@ -7231,7 +7219,7 @@ A или запись согласия covering
     <tu tuid="SessionsView.MetadataEditor.AccessProtocol.ELAR.U">
       <prop type="x-dynamic">true</prop>
       <tuv xml:lang="en">
-        <seg>all Users can access</seg>
+        <seg>all ordinary Users can access</seg>
       </tuv>
       <tuv xml:lang="ru">
         <seg>все пользователи могут получить доступ</seg>
```

Comment:
This and other translations of SessionsView.MetadataEditor.AccessProtocol.ELAR.U are inaccurate. I wonder if some effort to get an updated translation is worthwhile?